bin/aws-creds

```diff
@@ -164,15 +166,12 @@ when 'rm'
 
 when 'add-role'
   keychain = load_keychain
-  name = get_name_from_args_for_command('add-role')
-  item = get_valid_item(name)
-
-  role     = ask('         role name: ')
-  arn      = ask('          role arn: ')
+  role_name = ask('         role name: ')
+  arn       = ask('          role arn: ')
 
   keychain.generic_passwords.create(
-    label: "#{name} role #{role}",
```

Comment:
The (account) name isn't needed and prevents cross account roles.

Reply:
If I'm understanding this correctly, I have two concerns:

1. we're changing the existing api, orphaning existing roles saved in keychains
2. we're removing the "namespace" of an account name with a role name, creating the potential for conflicts if different accounts have the same role name.

Need to think a bit more about this, my first thought is maybe have an optional role-account argument on assume-role?

It doesn't seem like these changes are necessary for the console command.  Maybe splitting them up into two different PRs would help make things easier to review?